guides/non-tee.md

```diff
@@ -0,0 +1,245 @@
+
+Create the workload:
+```
+kubectl apply -f nginx.yaml
+```
+Output:
+```
+pod/nginx created
+```
+
+Ensure the pod was created successfully (in running state):
+```
+kubectl get pods
+```
+Output:
+```
+NAME    READY   STATUS    RESTARTS   AGE
+nginx   1/1     Running   0          3m50s
+```
+
+Now go back to the k8s node and ensure that you still don’t have any bitnami/nginx images on it:
+```
+root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
+```
+
+## Encrypted and/or signed images with attestation
+
+The previous example does not involve any attestation because the workload container isn't signed or encrypted
+and the workload itself does not require any secrets.
+
+This is not the case for most real workloads. It is recommended to use CoCo with signed and/or encrypted images.
+The workload itself can also request secrets from the attestation agent in the guest.
+
+Secrets are provisioned to the guest in conjunction with an attestation, which is based on hardware evidence.
+The rest of this guide focuses on setting up more substantial encrypted/signed workloads using attestation
+and confidential hardware.
+
+CoCo has a modular attestation interface and there are a few options for attestation.
+CoCo provides a generic Key Broker Service (KBS) that the rest of this guide will be focused on.
+The SEV runtime class uses `simple-kbs`, which is described in the [SEV guide](../guides/sev.md).
+There is also `eaa_kbc`/`verdictd` which is described [here](../guides/eaa_verdictd.md).
+
+### Select Runtime Class
+
+To use CoCo with confidential hardware, first switch to the appropriate runtime class.
+TDX has two runtime classes, `kata-qemu-tdx` and `kata-clh-tdx`. One uses QEMU as VMM and TDVF as firmware. The other uses Cloud Hypervisor as VMM and TD-Shim as firmware.
+
+For SEV(-ES) use the `kata-qemu-sev` runtime class and follow the [SEV guide](../guides/sev.md). 
+
+For SNP, use the `kata-qemu-snp` runtime class and follow the [SNP guide](../guides/snp.md).
+
+For `enclave-cc` follow the [enclave-cc guide](../guides/enclave-cc.md).
+
+### Deploy and Configure tenant-side CoCo Key Broker System cluster
+
+The following describes how to run and provision the generic KBS.
+The KBS should be run in a trusted environment. The KBS is not just one service,
+but a combination of several.
+
+A tenant-side CoCo Key Broker System cluster includes:
+- Key Broker Service (KBS): Brokering service for confidential resources.
+- Attestation Service (AS): Verifier for remote attestation.
+- Reference Value Provicer Service (RVPS): Provides reference values for AS.
+- CoCo Keyprovider: Component to encrypt the images following ocicrypt spec.
+
+To quick start the KBS cluster, a `docker compose` yaml is provided to launch.
+
+```shell
+# Clone KBS git repository
+git clone https://github.com/confidential-containers/kbs.git
```

Comment:
This is the old link of Trustee. We should use https://github.com/confidential-containers/trustee.git .

Reply:
Updated

Reply:
We also should change the `cd kbs/kbs` for `cd trustee/kbs` right?